pkg/analyser/analyzer.go

```diff
@@ -64,15 +63,19 @@ func (a Analyzer) Analyze(remoteResources, resourcesFromState []resource.Resourc
 				}
 				c := Change{Change: change}
 				c.Computed = a.isComputedField(stateRes, c)
+				if c.Computed {
+					haveComputedDiff = true
+				}
 				changelog = append(changelog, c)
 			}
 			if len(changelog) > 0 {
 				analysis.AddDifference(Difference{
 					Res:       stateRes,
 					Changelog: changelog,
 				})
-				a.sendAlertOnComputedField(stateRes, changelog)
 			}
+
+			a.sendAlertOnComputedField(haveComputedDiff)
```

Comment:
Maybe we do not need a dedicated function for this anymore.

It will be more straigthforward imho

```go
if haveComputedDiff {
  a.alerter.SendAlert("",
		  alerter.Alert{
			  Message: "You have diffs on computed fields, check the documentation for potential false positive drifts",
		  })
}
```

Reply:
:+1: